src/routes/contractViewState.js

```diff
@@ -0,0 +1,52 @@
+const warp = require('../warp');
+const { config } = require('../config');
+const { getLastStateFromDreCache, getCachedViewState, insertViewStateIntoCache } = require('../db/nodeDb');
+
+module.exports = async (ctx) => {
+  if (!config.availableFunctions.viewState) {
+    ctx.body = 'Contract view state functionality is disabled';
+    ctx.status = 503;
```

Comment:
is 503 correct here? I guess it usually means that server is not ready to handle the request (e.g. because it's down). axios library for example treats this code as a retriable. So maybe... 404 or 403?